src/client/components/RoomSelector.jsx

```diff
@@ -39,13 +39,11 @@ const RoomSelector = ({socket, joinRoom}) => {
       <h1>Public rooms</h1>
       <p>(click to join)</p>
       <div className="room-list">
-        {rooms.map(room => <div key={room.id}>
-          <span onClick={room.id !== currentRoomId ? (() => {
+        {rooms.map(room => <div key={room.id} onClick={room.id !== currentRoomId ? (() => {
```

Comment:
I'd rather keep this as a span, but the wrapping div is indeed pretty useless.

Reply:
Switched div to span.